webapp/src/components/data_selectors/jira_commentvisibility_selector/jira_commentvisibility_selector.tsx

```diff
@@ -23,31 +23,29 @@ type Props = BackendSelectorProps & {
     fieldName: string;
 };
 
-export default class JiraCommentVisibilitySelector extends React.PureComponent<Props> {
-    fetchInitialSelectedValues = async (): Promise<ReactSelectOption[]> => 
-        (!this.props.value || (this.props.isMulti && !this.props.value.length)) ? [] : this.searchCommentVisibilityFields('');
+export default function JiraCommentVisibilitySelector(props: Props) {
+    const fetchInitialSelectedValues = async (): Promise<ReactSelectOption[]> => 
+        (!props.value || (props.isMulti && !props.value.length)) ? [] : searchCommentVisibilityFields('');
```

Comment:
destructure things needed from `props` at a place and then use them directly instead of `props.<name>`
```
const {value, isMulti, searchCommentVisibilityFields} = props;
```

Reply:
Here we cannot use `searchCommentVisibilityFields` because we cannot redeclare block-scoped variable

Reply:
as discussed try changing the variable name on line 31